letters/modernized/14-letters.adoc

```diff
@@ -268,10 +286,13 @@ May it be so, says my soul.
 I write this in the innocency of my heart; make a right use of it,
 and let me hear from you, directed to H. G.
 
+[.signed-section-closing]
 With true love to you, and your honest wife, I rest your truly loving friend in the Lord,
```

Comment:
Ok, this one also goes onto two lines. With this one you could either put the whole thing up with the previous paragraph. OR you could put the first part with the paragraph above (i.e. With true love to you, and your honest wife,) and then the second part mark it as a SSCL (i.e. I rest your truly loving friend in the Lord,).  Hope that makes sense!!!

Reply:
Yes, that makes total sense, thanks!